src/django_linear_migrations/management/commands/makemigrations.py

```diff
@@ -8,8 +9,15 @@
 
 
 class Command(BaseCommand):
-    def write_migration_files(self, changes: dict[str, list[Migration]]) -> None:
-        super().write_migration_files(changes)
+    def write_migration_files(
+        self,
+        changes: dict[str, list[Migration]],
+        update_previous_migration_paths: dict[str, str] | None = None,
+    ) -> None:
+        if django.VERSION >= (4, 2, 0):
```

Comment:
equivalent, and later [removable by django-upgrade](https://github.com/adamchainz/django-upgrade#versioned-blocks):

```suggestion
        if django.VERSION >= (4, 2):
```